modules/iam/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/acls/Path.scala

```diff
@@ -179,6 +186,9 @@ object Path {
 
     override def expand: Set[Path] = tail.expand + this
 
+    override def startsWith(path: Path): Boolean =
+      expand.toSeq.startsWith(path.expand.toSeq)
```

Comment:
If `expand` returns a `Set` then there's no particular guarantees about ordering, and there's no guarantee that `path.expand.toSeq` will be in any particular order, so it might not always work.